components/Price/index.jsx

```diff
@@ -21,23 +22,38 @@ import styles, { discounted } from './style';
  * @return {JSX}
  */
 const Price = (props) => {
-  let localStyles = styles;
-  localStyles += props.discounted ? ` ${discounted}` : '';
+  const containerClasses = [
+    styles.container,
+    props.className,
+    ...props.discounted && [styles.discounted],
+  ].join(' ');
 
   /**
    * A unitPriceMin > 0 means, that the product has child products with different prices.
    * The unitPriceMin contains the lowest of these prices and will be
    * displayed with a 'From' prefix.
    */
   return (
-    <div className={`${localStyles} ${props.className}`}>
+    <div className={containerClasses}>
       {props.unitPriceMin ? (
         <I18n.Text string="price.from">
-          <I18n.Price forKey="price" price={props.unitPriceMin} currency={props.currency} fractions={props.fractions} />
+          <I18n.Price
+            currency={props.currency}
+            forKey="price"
+            fractions={props.fractions}
+            price={props.unitPriceMin}
+          />
         </I18n.Text>
       ) : (
-        <I18n.Price price={props.unitPrice} currency={props.currency} fractions={props.fractions} />
-      )}
+        <I18n.Price
+          currency={props.currency}
+          fractions={props.fractions}
+          price={props.unitPrice}
+        />
+        )}
+        {props.taxDisclaimer && showTaxDisclaimer ? (
```

Comment:
wrong indentation reported by ESlint. Please check all changed files again
seems your IDE is not set-up correctly